cfgov/jinja2/v1/_layouts/content-base-sidebar-first.html

```diff
@@ -3,7 +3,10 @@
 {% block pre_content scoped -%}
     {# TODO: Remove non-wagtail logic once old pages have been converted #}
     {% if page %}
+        {% if not breadcrumb_items %}
         {% set breadcrumb_items = page.get_breadcrumbs(request) %}
+        {% endif %}
+        
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/3006/commits/33f77677f3e9792e2e46cf1d352584ea49991e96